### PR TITLE
Created Admin UI

### PR DESCRIPTION
**Pull request by ElinorW**
Administrator's views with static content. No database connection.

### CODE REVIEW

**Review by titusgicheru — CHANGES_REQUESTED** [review]
- Please have **OfficeBooks** and **OfficeItems** in their own controllers Views
- While the UI looks great on the Desktop browser, consider making it responsive. Tables may not be the best approach to achieve responsiveness.

**Review by irvinesunday — COMMENTED** [review]
isbn toggle

I can't think of any use case for needing to have the `Book ISBN` toggle-able to display items in ascending or descending order (based on their ISBN). I think we can remove this one feature. Same applies to the Office Items `Description` as shown below.

description

**Comment by irvinesunday** [comment]
responsive_tables

@titusgicheru I think the tables handle responsiveness quite okay in different window sizes. It's just the `Borrowed Items` and `Borrowed Books` table that struggle a little when their window sizes are pushed to their minimum size. @ElinorW please check this out.

File: AssetTracking/AssetTracking/Views/Home/OfficeBooks.cshtml [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+                <!--Table body-->
+                <tbody>
+                    <tr>
+                        <td>1234</td>
+                        <td>Cracking the coding Interview</td>
+                        <td>Otto</td>
+                        <td>Elinor Kaguongo</td>
+                        <td>1/7/2019</td>
+                        <td>19/7/2019</td>
+                        <td><a asp-controller="Home" asp-action="ViewBookLog"> View Book Log </a>|
```
Comment (irvinesunday) [inline AssetTracking/AssetTracking/Views/Home/OfficeBooks.cshtml]: `View History` sounds more descriptive.

File: AssetTracking/AssetTracking/Views/Home/OfficeBooks.cshtml [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+                <!--Table body-->
+                <tbody>
+                    <tr>
+                        <td>1234</td>
+                        <td>Essential C# 7.0 (6th Edition) </td>
+                        <td>Otto</td>
+                        <td>
+                            <a asp-controller="Home" asp-action="UpdateBookDetails"> Edit </a>|
+                            <a asp-controller="Home" asp-action="DeleteBookDetails"> Delete</a>
+                        </td>
```
Comment (irvinesunday) [inline AssetTracking/AssetTracking/Views/Home/OfficeBooks.cshtml]: Can we also add a `View Details` link here for each of the list of Book Items.

**Comment by irvinesunday** [comment]
Consider adding the following:
- **Name** of the signed-in user and role (e.g Administrator)
- Quick snapshot of the running **total** of the items in each table category for both Office Books and Office Items
- An **overdue warning icon** (e.g. an exclamation mark) next to overdue items/books just before the `Send Reminder` button as below. 
reminder

**Comment by titusgicheru** [comment]
> responsive_tables
> 
> @titusgicheru I think the tables handle responsiveness quite okay in different window sizes. It's just the `Borrowed Items` and `Borrowed Books` table that struggle a little when their window sizes are pushed to their minimum size. @ElinorW please check this out.

@irvinesunday I think if we want to achieve responsiveness in the UI,  using tables is generally not a good idea. It's better not to use tables unless we explicitly want data to be laid out in rows and columns and in this scenarios we are most likely going to trade off responsiveness.

**Comment by ElinorW** [comment]
I've updated the files. 
Made the requested changes, still working on adding the following
- Name of the signed-in user and role (e.g Administrator)
- An overdue warning icon (e.g. an exclamation mark) next to overdue items/books just before the send reminder button.

**Pull request merged**